lab-cameron/src/components/search-result-list.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const SearchForm = ({ hasSearched, loading, noResults, topics }) => {
```

Comment:
this should be a component, not a function. 


class SearchResults extends React.Component